stat/roc.go

```diff
@@ -126,3 +126,32 @@ func ROC(cutoffs, y []float64, classes []bool, weights []float64) (tpr, fpr, thr
 
 	return tpr, fpr, cutoffs
 }
+
+// TOC returns the Total Operating Characteristic for the classes provided,
+// which should be a boolean variable ordered by a rank variable. TOC also
+// returns the minimum and maximum bounds for the TOC.
+//
+// More details about TOC curves are available at
+// https://en.wikipedia.org/wiki/Total_operating_characteristic
+func TOC(classes []bool) (min, toc, max []float64) {
```

Comment:
It's a technicality, but I would call the toc' output something like ntp' for number of true positives, as the "TOC" refers to the curve, y- and x- coordinates, while this slice is just the y-coordinates. I don't feel strongly about it though, `toc' is fine too.

Reply:
Done